metadata-ingestion/src/datahub/ingestion/source/elastic_search.py

```diff
@@ -458,6 +520,39 @@ def _extract_mcps(
                 ),
             )
 
+        if self.source_config.profiling.enabled:
+            cat_response = self.client.cat.indices(
+                index=index, params={"format": "json", "bytes": "b"}
+            )
+            if len(cat_response) == 1:
+                index_res = cat_response[0]
+                docs_count = int(index_res["docs.count"])
+                size = int(index_res["store.size"])
+                if len(self.source_config.collapse_urns.urns_suffix_regex) > 0:
```

Comment:
why is `collapse_urns` required for profiling?

also need to document how collapse_urns interacts with profiling - it seems to merge the results together?